hbase-server/src/test/java/org/apache/hadoop/hbase/replication/TestReplicationBase.java

```diff
@@ -238,26 +243,27 @@ private static void startClusters() throws Exception {
     // as a component in deciding maximum number of parallel batches to send to the peer cluster.
     UTIL2.startMiniCluster(NUM_SLAVES2);
 
+    connection1 = ConnectionFactory.createConnection(CONF1);
+    connection2 = ConnectionFactory.createConnection(CONF2);
     admin = new ReplicationAdmin(CONF1);
```

Comment:
This was removed @apurtell. It was just one line change. 
`admin = new ReplicationAdmin(CONF1);`